data/settings.js

```diff
@@ -10272,6 +10272,17 @@ The most common choices are \`commonName\` and \`x500UniqueIdentifier\`.
 Note: [[setting,auth_ssl_username_from_cert]] MUST be enabled.`
 	},
 
+	ssl_peer_certificate_fingerprint_hash: {
```

Comment:
ssl_server_request_client_cert setting has now `any-cert` value, needs to be documented.